### PR TITLE
Code cleanup 2

### PR DESCRIPTION
**Pull request by Tobi-Sheriff**
Added more test cases to my APIs

### CODE REVIEW

File: commentRepository.js [hunk 1]
```diff
@@ -6,26 +6,33 @@ class CommentRepository {
...
-    // Retrieve movie from data storage
     return this.dataStorage.initializeComments();
   }
 
   async getComments() {
     return this.dataStorage.getComments();
   }
   async getCommentsByMovieId(id) {
-    const comments = await this.dataStorage.getComments();
+    const comments = await this.dataStorage.getComments();
```
Comment (ajibigad) [inline commentRepository.js:16]: What changed here?

File: commentRepository.js [hunk 2]
```diff
@@ -6,26 +6,33 @@ class CommentRepository {
...
-    const comments = await this.dataStorage.getComments();
+    const comments = await this.dataStorage.getComments();    
     return comments.response.filter((comment) =>
       comment.movie_id === parseInt(id)
     );
   }
 
-  async addComment(comment) {
-    return await this.dataStorage.addComment(comment);
+  async addComment(id, content, author) {
```
Comment (ajibigad) [inline commentRepository.js:22]: If this is the movie id, i think you should name it movie id so it is not confused as the comment id

File: commentRepository.js [hunk 3]
```diff
@@ -6,26 +6,33 @@ class CommentRepository {
...
+    const newComment = {
+      id: 11,
```
Comment (ajibigad) [inline commentRepository.js:24]: You want to always hard code the comment id?

**Review by ajibigad — APPROVED** [review]
Makes improvements we discussed in the next PR

File: commentRepository.js (hunk 1 above)
Comment (Tobi-Sheriff) [reply commentRepository.js:16]: I can't exactly recall. The change is probably more clear in the first code-cleanup i pushed but i think i was returning the data upon calling it initialy before updating it to storing in a variable and performing some actions with it

File: commentRepository.js (hunk 2 above)
Comment (Tobi-Sheriff) [reply commentRepository.js:22]: Ok noted, will do that

File: commentRepository.js (hunk 3 above)
Comment (Tobi-Sheriff) [reply commentRepository.js:24]: Uhm of course not, I would update that.

**Pull request merged**